bash/force-level1-csd.sh

```diff
@@ -186,7 +186,11 @@ SENTINEL=0
 UPDATE=0
 KEEPMETA=0
 
-ARGS=`getopt -o c:d:nhks:t:u --long cloudcover:,daterange:,no-act,help,keep-meta,sensors:,tier:,update -n $0 -- "$@"`
+# change - to %dummy% if followed by integer: prevent interpretation as option by getopt
+ARGS=$(echo "$@" | sed -E "s/-([0-9])/%dummy%\1/g")
```

Comment:
This way, all hyphens will be replaced, even when in one of the filenames.

shouldn't there be a space before "-":
"s/ -([0-9])/%dummy%\1/g"

Only the critical hyphens will then be replaced.

Reply:
Thanks, fixed!